ucsb-cs184-f24/team08-StudyMeets
# Feature/hw04 dark theme

Addresses #72 

Acceptance Criteria: Adds a toggle slider to the Settings tab that allows users to change the UI theme throughout the app to a dark theme.

Testing:
1. Navigate to Settings
2. Toggle the `Dark Mode` slider under `General Settings`
3. Verify screen changes to a dark theme
4. Verify all UI throughout the app is in a dark theme (excluding any WIP pages such as Notification Settings, Friends, etc)

Additional notes:
All the colors used for each theme are defined in theme/themes.js

Screenshots:
![image](https://github.com/user-attachments/assets/d7dd980f-5179-45c2-a2b8-9948680370c1)
![image](https://github.com/user-attachments/assets/553577ed-b22b-4f13-bdda-7e1f5e4b054a)
![image](https://github.com/user-attachments/assets/20d3b474-8425-4ad9-9a1f-4d81f006fa0f)
